### ISSUE TITLE
name 'WideDeep' is not defined

### ISSUE THREAD

**Issue by egmaminta** [jrzaurin/pytorch-widedeep#63]
Good day. I'm testing pytorch-widedeep on tabular data (specifically TabResnet). I followed the steps as shown here in this article: https://towardsdatascience.com/pytorch-widedeep-deep-learning-for-tabular-data-9cd1c48eb40d. When I attempted to run the code to instantiate WideDeep, an error shows saying WideDeep is not defined. I can't seem to find any workaround on this one. I was wondering if you could enlighten me on this. (Still new to this, sorry!) Thanks!

**Comment by egmaminta** [comment]
> Good day. I'm testing pytorch-widedeep on tabular data (specifically TabResnet). I followed the steps as shown here in this article: https://towardsdatascience.com/pytorch-widedeep-deep-learning-for-tabular-data-9cd1c48eb40d. When I attempted to run the code to instantiate WideDeep, an error shows saying WideDeep is not defined. I can't seem to find any workaround on this one. I was wondering if you could enlighten me on this. (Still new to this, sorry!) Thanks!

UPDATE: So I git cloned this repo, and copied the wide_deep.py file. I then used:

from wide_deep import WideDeep

The error vanished but I'm not quite sure I'm doing it right. Anyone who can confirm on this? Thanks!

**Comment by jrzaurin** [comment]
Hey @egmaminta 

no, you are not doing it right. You have two options. 

you can install the library via pip: 

```
pip install pytorch-widedeep
```

alternatively, you can install it from github for you to get the latest version

```
pip install git+https://github.com/jrzaurin/pytorch-widedeep.git
```

if you want to edit it, you can do:

```
# Clone the repository
git clone https://github.com/jrzaurin/pytorch-widedeep
cd pytorch-widedeep

# Install in dev mode
pip install -e .
```

Then you can check the updated [docs](https://pytorch-widedeep.readthedocs.io/en/latest/index.html) and [examples](https://github.com/jrzaurin/pytorch-widedeep/tree/master/examples).

finally, whichever way you have chosen to install it, the `WideDeep` class is available in the `models` module:

```python
from pytorch_widedeep.models import WideDeep
```

But please, have a look to the docs and the examples

Also stay tuned for the new features coming in v2 (many of them) and join the slack channel! 😉: https://join.slack.com/t/pytorch-widedeep/shared_invite/zt-soss7stf-iXpVuLeKZz8lGTnxxtHtTw

Talk soon!

**Comment by egmaminta** [comment]
Hi! Although I did install the library, the error would persist. I might try it again later. Looking forward to upcoming features. (Can you also include step-by-step guides in defining optimizers, learning rate schedulers?) Thanks and all the best!

**Comment by jrzaurin** [comment]
Remember you imported as: 

```
from wide_deep import WideDeep
```

and should be

```
from pytorch_widedeep.models import WideDeep
```
